composer.json

```diff
@@ -5,7 +5,7 @@
     "type": "library",
     "require": {
         "php": ">=7.0",
-        "illuminate/support": "^5.6",
+        "illuminate/support": "^5.7|^6.0|^7.0|^8.0",
```

Comment:
Looking into this more from Laravel 7 onwards support for monolog 1 was dropped. https://github.com/laravel/framework/pull/29866. We will need another separate release for Laravel 7/8 using Monolog 2 otherwise we can't install dependencies from Laravel 7+

Reply:
From discussions, to prevent having to support multiple releases, we are only going to be supporting L6+ moving forward (which is the earliest that support Monolog 2) so need to make sure we use Monolog 2 instead and test this still works as expected. See https://github.com/laravel/framework/blob/43518afa5253600513b19ba7c3d4aad2a3eb69c3/composer.json#L27.

We may also need to update our version of testbench to support this.